importer/RoRo.py

```diff
@@ -1,29 +1,117 @@
+        the same.
+        """
+        if self.has_non_empty_attribute("localitate"):
+            loc_item = None
+            if utils.count_wikilinks(self.localitate) > 0:
+                loc_link = utils.get_wikilinks(self.localitate)[0]
+                loc_item = utils.q_from_wikipedia("ro", loc_link.title)
+                adm_item = self.get_statement_values("located_adm")
+                if loc_item and loc_item != adm_item[0]:
+                    self.add_statement("location", loc_item)
+
+            if not loc_item:
+                self.add_to_report("localitate", self.localitate, "location")
+
+    def set_wlm_id(self):
+        """
+        Set the WLM ID.
+
+        Do _not_ use the Romania LMI code (P1770)
+        as it's made up and has no relevance to the
+        real world.
+        """
+        ccode = self.mapping["country_code"].upper()
+        wlm_id = "{}-{}".format(ccode, self.cod)
+        self.add_statement("wlm_id", wlm_id)
+
+    def update_descriptions(self):
+        adm_code = self.judetul_iso
+        counties = self.data_files["counties"]
+        county_item = utils.get_item_from_dict_by_key(dict_name=counties,
+                                                      search_term=adm_code,
+                                                      return_content_of="itemLabel",
+                                                      search_in="iso_code")
+        if len(county_item) == 1:
+            place_name = "{}, Romania".format(county_item[0])
+        else:
+            place_name = "Romania"
+        desc = "heritage site in {}".format(place_name)
+        self.add_description("en", desc)
+        self.add_disambiguator(utils.remove_markup(str(self.cod)))
+
+    def set_address(self):
+        if self.has_non_empty_attribute("adresa"):
+            adr_lower = self.adresa.lower()
+            adr_nice = utils.remove_markup(self.adresa)
+            if "piața" in adr_lower or "str." in adr_lower:
+                if self.has_non_empty_attribute("localitate"):
+                    town = utils.remove_markup(self.localitate)
+                    adr_nice = "{}, {}".format(adr_nice,
+                                               town)
+                self.add_statement("located_street", adr_nice)
+            else:
+                directions = utils.package_monolingual(adr_nice, 'ro')
+                self.add_statement("directions", directions)
+
+    def update_labels(self):
+        romanian = utils.remove_markup(self.denumire)
+        self.add_label("ro", romanian)
+
+    def set_monuments_all_id(self):
+        """Map which column name in specific table to  ID in monuments_all."""
+        self.monuments_all_id = str(self.cod)
+
+    def __init__(self, db_row_dict, mapping, data_files, existing, repository):
+        Monument.__init__(self, db_row_dict, mapping,
+                          data_files, existing, repository)
+        self.set_monuments_all_id()
```

Comment:
give "cod" as a parameter and you can drop the local method

Reply:
Done.